frontend/src/components/simple-auto-filter/simple-auto-filter.tsx

```diff
@@ -80,7 +90,64 @@ const SimpleAutoFilter: FC = () => {
     navigate(API.SEARCH);
   };
 
-  if (isLoading) return <h1>Loading...</h1>;
+  const brandsOptions = useMemo(
+    () =>
+      brands?.map(
+        (item) =>
+          ({
+            label: item.name,
+            id: item.id,
+          } as AutocompleteValueType),
+      ),
+    [brands],
+  );
+
+  const modelsOptions = useMemo(
+    () =>
+      models?.map(
+        (item) =>
+          ({
+            label: item.name,
+            id: item.id,
+          } as AutocompleteValueType),
+      ),
+
+    [models],
+  );
+
+  const selectedBrandName = useMemo(() => {
+    return getValueById(brands || [], brandId);
+  }, [brandDetails[0]]);
+
+  const selectedModelName = useMemo(
+    () => getValueById(models || [], modelIds[0]),
+    [modelIds[0]],
+  );
```

Comment:
Arrays indexing is hard to read

Reply:
I I simplified where I could. In this case I show only first selected model in the simple form.